content/vault/v1.20.x/content/docs/deploy/kubernetes/comparisons.mdx

```diff
@@ -0,0 +1,138 @@
+The provider runs Vault Agent alongside the CSI driver for authentication and secret retrieval.
+
+**Best for:** Organizations using multiple secret stores in addition to Vault, or those who prefer vendor-neutral CSI standards.
+
+## Vault Agent Injector
+
+The [Vault Agent Injector](/vault/docs/platform/k8s/injector) injects Vault Agent sidecar containers into pods. 
+The Agent containers authenticate with Vault and render secrets to a shared memory volume for consumption by application containers.
+
+**Best for:** Applications requiring dynamic secret rotation or direct Vault Agent functionality.
+
+
+## Use case recommendations
+
+The appropriate integration for your environment depends on the applicable
+security policies, secret data governance, ease of use requirements, and
+system availability guarantees.
+
+| Limitation                                        | Vault Secrets Operator            | Vault Secrets Store CSI provider　| Vault Agent Injector
+| ------------------------------------------------- | --------------------------------- | ---------------------------- | --------------
+| Simple configuration                              | Yes                               | Yes                          | No
+| Low resource consumption. I/O, CPU, Memory        | Yes                               | Yes                          | No
+| Secret data limited to ephemeral volumes          | No, except with CSI driver        | Yes                           | Yes
+| Pod Autoscaling dependent on Vault availability   | No                                | Yes                          | Yes
+| Adoption of the K8s operator model                | Yes                               | Yes                          | No
+| Personas easily separated via K8s [RBAC] policies | Yes                               | No                           | No
+
+[RBAC]: https://kubernetes.io/docs/reference/access-authn-authz/rbac
+
+
+## Performance considerations
+
+Performance characteristics vary significantly between integrations based on
+their architectural patterns.
+
+<Tabs>
+	
+<Tab heading="Load on Vault">
+
+- **Vault Secrets Operator** - Lowest load due to per-CRD connections and cluster-local caching.
+- **Vault Secrets Store CSI provider** - Higher load due to per-pod connections.
+- **Vault Agent Injector** - Highest load due to per-pod connections and sidecar pattern.
+	
+</Tab>
+	
+<Tab heading="Secret availability">
+
+Integrations that use ephemeral storage have variable availability because
+secret access depends on Vault availability.
+
+- **Vault Secrets Store CSI provider** - Variable availability.
+- **Vault Agent Injector** - Variable availability.
+- **Vault Secrets Operator** - Best availability. Kubernetes Secrets act as durable cache.
```

Comment:
> secret access depends on Vault availability.

Isn't that true for any of these? i.e. nothing can fetch secrets if Vault is down. If a secret is fetched from Vault and rendered, Vault subsequently going down isn't going to clear the rendered secret for CSI provider or Agent, is it?

Integrations using ephemeral storage means the rendered secret lifecycle is tied to the Pod lifecycle, instead of being independent for VSO, maybe re-frame along those lines?